claudia-lab/server.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+//node modules
+const http = require('http');
+const url = require('url');
+const queryString = require('querystring');
+
+// npm modules
+const cowsay = require('cowsay');
+
+// app moduels
+const parseBody = require('./lib/parsebody.js');
+
+// module constats
+const PORT = process.env.PORT || 3000;
+
+//creates server and takes a callback that handles the request and sends a response
+const server = http.createServer(function(req, res) {
+  req.url = url.parse(req.url); //gets url
+  req.url.query = queryString.parse(req.url.query); //gets query string
+
+  if (req.url.pathname === '/') {
+    res.writeHead(200, {'Content-Type': 'text/plain'});
+    res.write('Hello World');
+    res.end();
+    return;
+  }
+
+  if (req.method === 'GET' && req.url.pathname === '/cowsay') {
+    if (req.url.query.text) {
+      res.writeHead(200, {'Content-Type': 'text/plain'});
```

Comment:
@loomnugget as odd as it may seem, best practice is to break this down into new lines - for example:
```
res.writeHead(200, {
  'Content-Type': 'text/plain'
});
```